app/controllers/insured/families_controller.rb

```diff
@@ -24,7 +24,7 @@ class Insured::FamiliesController < FamiliesController
   around_action :cache_hbx, only: [:home]
 
   def home
-    authorize @family, :show?
+    authorize @family, :legacy_show?
```

Comment:
why we just dont replace :legacy_show with :show?

Reply:
We have not implemented methods for the SHOP and FEHB markets yet. As part of the next PR, we will be implementing these methods and once we do that that is when we will delete the `:legacy_show?` method and use the new `FamilyPolicy#show?`. This way we will continue to develop on Green build.